src/Poyii/Informix/InformixDBServiceProvider.php

```diff
@@ -21,11 +21,9 @@ class InformixDBServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        $this->publishes(
-            [
-                __DIR__.'/../../config/informix.php' => config_path('informix.php'),
-            ]
-        );
+        $this->publishes([
+            __DIR__.'/../../config/informix.php' => config_path('informix.php'),
+        ], 'informix-config');
```

Comment:
Keep the old line break, and add the tag as a new line, to the array.

Reply:
fixed as in [31cb688](https://github.com/isoc-il/laravel-ifx/pull/10/commits/31cb68872bbd37a3ee3f3754df395cb746c15bf8)